test/react/jsx/jsx_prepocessor_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'test_helper'
+
+when_sprockets_available do
+  class JSXPreprocessorTest < ActionDispatch::IntegrationTest
+    EXPECTED_JS = <<javascript
+React.createElement(\"div\", { className: \"le-javascript-child\" });
+React.createElement(\"div\", { className: \"le-javascript\" });
+javascript
+
+    test 'accepts harmony: true option' do
+      get '/assets/require_test/jsx_preprocessor_test.js'
+      assert_response :success
+      assert_match(EXPECTED_JS, @response.body)
```

Comment:
seems like this is failing on whitespace :weary: 

maybe you could move this custom assertion to `test_helper` and use it here? https://github.com/reactjs/react-rails/blob/master/test/react/jsx_test.rb#L71-L77